code/cryptography/vigenere_cipher/vigenere_cipher.c

```diff
@@ -1,59 +1,83 @@
+// Vigenere Cipher
+
+// Author: Rishav Pandey
+
 #include <stdio.h>
-#include <ctype.h>
+//#include <cs50.h>
 #include <string.h>
+#include <stdlib.h>
+#include <ctype.h>
 
-int isalphastr (char*);
-
-int main (void)
+int main(int argc, char* argv[])
 {
-  char k[100] = "ovaltine";
-  char p[100] = "Meet me at 11:00 AM sharp.";
-  
-    if(!isalphastr(k))
+    //check if user give more or less than two arguments.
```

Comment:
Explicitly defining the key and message makes the code easier to understand. So, changes to this part of code is not necessary 👍

Reply:
this is not my code.
I have completely changed vigenere code and implemented my new one.